5arrays.html

```diff
@@ -49,54 +49,49 @@
           </tr>
         </thead>
         <tbody id="tbody">
-          
+
         </tbody>
       </table>
-      
+
       <script>
         var sourceData1 = [ {id: 1, name: "piyush"},
                             {id: 2, name: "sourabh"},
                             {id: 3, name: "shivam"},
                             {id: 4, name: "rohit"},
-                            {id: 14, name: "rohit"}];
+                            {id: 14, name: "rohit"} ];
         var sourceData2 = [ {id: 4, age: 21},
                             {id: 3, age: 33},
                             {id: 2, age: 12},
-                            {id: 1, age: 23}, 
+                            {id: 1, age: 23},
                             {id: 5, age: 24},
-                            {id: 11, age: 23}, 
-                            {id: 51, age: 24}
-                   
-                   ];
-                   
-        
+                            {id: 11, age: 23},
+                            {id: 51, age: 24} ];
+
+
         var destinationData = [];
         for(var i = 0; i < sourceData1.length; i++) {
             for(var j=0; j < sourceData2.length; j++) {
                 if(sourceData1[i].id == sourceData2[j].id) {
                     destinationData.push( {id: sourceData1[i].id, name: sourceData1[i].name, age: sourceData2[j].age});
                     sourceData2.splice(j, 1);
-                } 
+                    break;
+                }
             }
         }
-        console.log(sourceData2);
-        console.log(destinationData);
+
         destinationData = destinationData.concat(sourceData2);
-        console.log("destinationData array after change");
-        console.log(destinationData);
-        
+
         var data= "";
```

Comment:
Improve the readability of code. Operator used must have space before and after it.